plugins/osx/osx.plugin.zsh

```diff
@@ -199,6 +208,7 @@ EOF
 			echo "\tnext|previous\tplay next or previous track"
 			echo "\tshuf|shuffle [on|off|toggle]\tSet shuffled playback. Default: toggle. Note: toggle doesn't support the MiniPlayer."
 			echo "\tvol\tSet the volume, takes an argument from 0 to 100"
+			echo "\tplaying\tShow what song is currently playing in iTunes."
```

Comment:
Could replace this whole series of `echo`s with ` <<EOF` here-document like the other usage sections. Would be more readable.